cfme/tests/cloud_infra_common/test_relationships.py

```diff
@@ -390,3 +396,72 @@ def test_change_network_security_groups_per_page_items():
         1524443
     """
     pass
+
+
+@test_requirements.relationships
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.meta(coverage=[1729953])
+def test_datastore_relationships():
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Cloud
```

Comment:
**Required** Casecomponent should be Infra as you are adding infra provider in step 1 below.